backend/api.py

```
@@ -1509,22 +1513,31 @@ async def open_file(body: dict, request: Request, _=Depends(verify_local_request
     try:
         import subprocess
         import platform
         
+        logger.info(f"[DEBUG-OPEN] Platform: {platform.system()}")
         if platform.system() == 'Windows':
+            logger.info("[DEBUG-OPEN] Calling os.startfile...")
             os.startfile(file_path)
+            logger.info("[DEBUG-OPEN] os.startfile completed successfully")
         elif platform.system() == 'Darwin':  # macOS
+            logger.info("[DEBUG-OPEN] Calling open via subprocess...")
             subprocess.run(['open', file_path])
```

Ruff (0.15.14)

[warning] Async functions should not run processes with blocking methods

(ASYNC221)